Sources/NIO/ByteBuffer-views.swift

```diff
@@ -69,8 +182,14 @@ public struct ByteBufferView: RandomAccessCollection {
 }
 
 extension ByteBuffer {
-    /// A view into the readable bytes of the `ByteBuffer`.
+    /// A view into the bytes of the `ByteBuffer`.
+    @available(*, deprecated, renamed: "view")
```

Comment:
I don't think we should rename `readableBytesView`. What's wrong with just leaving this as is?